lib/Rides.dart

```diff
@@ -313,67 +310,92 @@ class RideInProgressCard extends StatelessWidget {
       },
       child: DecoratedBox(
           decoration: BoxDecoration(
-              color: selected ? Color(0xFFBDBDBD) : Colors.white,
+              color: selected ? Color.fromRGBO(167, 167, 167, 0.4) : Colors.white,
               borderRadius: BorderRadius.all(Radius.circular(8)),
               boxShadow: [
-                BoxShadow(
-                    blurRadius: 2,
-                    spreadRadius: 0,
-                    color: Colors.black.withOpacity(0.25))
+                CarriageTheme.shadow
               ]),
-          child: Container(
-            padding: const EdgeInsets.all(16),
-            child: Column(
-                mainAxisSize: MainAxisSize.max,
-                crossAxisAlignment: CrossAxisAlignment.start,
-                children: [
-                  selected
-                      ? Icon(Icons.check_circle, size: 20, color: Colors.black)
-                      : Container(
-                          width: 20,
-                          height: 20,
-                          decoration: BoxDecoration(
-                              shape: BoxShape.circle, color: Color(0x7FC4C4C4)),
+          child: Stack(
+            children: [
+              Padding(
+                padding: EdgeInsets.only(left: 8, top: 8),
+                child: selected
+                    ? Icon(Icons.check_circle, size: 20, color: Colors.black)
+                    : Container(
+                  width: 20,
+                  height: 20,
+                  decoration: BoxDecoration(
+                      shape: BoxShape.circle, color: Color(0x7FC4C4C4)),
+                ),
+              ),
+              Container(
+                padding: const EdgeInsets.all(16),
+                child: Column(
+                    mainAxisSize: MainAxisSize.max,
+                    children: [
+                      Center(
+                        child: CircleAvatar(
+                          radius: 16,
+                          backgroundImage: AssetImage('assets/images/terry.jpg'),
+                        ),
+                      ),
+                      SizedBox(height: 8),
+                      Center(
+                          child: Text(ride.rider.firstName,
+                              style: Theme.of(context).textTheme.subtitle2
```

Comment:
Same here for not using the Carriage theme

Reply:
Fixed